src/Core/Graphics/LevelSprite.cpp

```diff
@@ -486,6 +492,10 @@ namespace obe::Graphics
         const int zdepth = data.contains(vili::NodeType::DataNode, "z-depth")
                                ? data.getDataNode("z-depth").get<int>()
                                : 1;
+        
+        const bool antiAliasing = data.contains(vili::NodeType::DataNode, "antiAliasing")
+                               ? data.getDataNode("antiAliasing").get<bool>()
+                               : true;
```

Comment:
It could be nice to have a workspace-wide antialiasing settings instead of this `true`